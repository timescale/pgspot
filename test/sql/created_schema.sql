CREATE SCHEMA test_schema;

CREATE COLLATION IF NOT EXISTS test_schema.collation3 (locale = 'fr_FR.utf8');
CREATE FOREIGN TABLE IF NOT EXISTS test_schema.unsafe_table4(c4444 text) SERVER server4;
CREATE MATERIALIZED VIEW IF NOT EXISTS test_schema.matview6 AS SELECT pg_catalog.now();
CREATE SEQUENCE IF NOT EXISTS test_schema.sequence8;
CREATE TABLE IF NOT EXISTS test_schema.table10(c10 text);
CREATE TABLE IF NOT EXISTS test_schema.table11 AS SELECT pg_catalog.now();
