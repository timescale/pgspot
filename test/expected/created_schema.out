
Errors: 0 Warnings: 0 Unknown: 0
